MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,11 +2,163 @@ import UIKit
+        QuizQuestion(
+            image: "Old",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false),
+        QuizQuestion(
+            image: "The Ice Age Adventures of Buck Wild",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false),
+        QuizQuestion(
+            image: "Tesla",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false),
+        QuizQuestion(
+            image: "Vivarium",
+            text: "Рейтинг этого фильма больше чем 6?",
+            correctAnswer: false)
+    ]
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        let currentQuestion = questions[currentQuestionIndex]
+        let convertVar = convert(model: currentQuestion)
+        show(quiz: QuizStepViewModel(image: convertVar.image, question: convertVar.question, questionNumber: convertVar.questionNumber))
+        
+    }
+    
+    @IBAction private func yesButton(_ sender: Any) {
+        if questions[currentQuestionIndex].correctAnswer == true{
+            showAnswerResult(isCorrect: true)
+        } else {
+            showAnswerResult(isCorrect: false)
+        }
+    }
+    
+    @IBAction private func noButton(_ sender: Any) {
+        if questions[currentQuestionIndex].correctAnswer == true{
+            showAnswerResult(isCorrect: false)
+        } else {
+            showAnswerResult(isCorrect: true)
+        }
+    }
+    private func show(quiz step: QuizStepViewModel) {
+        // здесь мы заполняем нашу картинку, текст и счётчик данными
+        imageView.image = step.image
+        counterLabel.text = step.questionNumber
+        textLabel.text = step.question
+    }
+    
+    private func show(quiz result: QuizResultsViewModel) {
+        let alert = UIAlertController(title: result.title,
+                                      message: result.text,
+                                      preferredStyle: .alert) // preferredStyle может быть .alert или .actionSheet
+
+        // создаём для него кнопки с действиями
+        let action = UIAlertAction(title: "Сыграть еще раз!", style: .default) { _ in
+            self.currentQuestionIndex = 0
+            
+          //заново показываем первый вопрос
+            let firstQuestion = self.questions[self.currentQuestionIndex]
+            let viewModel = self.convert(model: firstQuestion)
+            self.show(quiz: viewModel)
+        }
+        correctAnswers = 0
+        alert.addAction(action)
+        // показываем всплывающее окно
+        self.present(alert, animated: true, completion: nil)
+    }
+    
+    private func convert(model: QuizQuestion) -> QuizStepViewModel {
+        return QuizStepViewModel(
+            image: UIImage(named: model.image) ?? UIImage(),
+            question: model.text,
+            questionNumber: "\(currentQuestionIndex + 1)/\(questions.count)")
+    }
+    private func showAnswerResult(isCorrect: Bool) {
```

Comment:
между функциями обычно оставляют одну пустую строчку
```suggestion

    private func showAnswerResult(isCorrect: Bool) {
```

Reply:
Понял, принял, постараюсь запомнить, спасибо!